apps/site/pages/components/molecules/region-bar.mdx

```diff
@@ -0,0 +1,161 @@
+---
+description: The Region Bar is a similar proposal for the Region Button, usually used on mobile devices only. It triggers the RegionModal.
+keywords:
+  - UI library
+  - ecommerce
+  - react
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Button___6d6b16886d5d7e2a05dba8c45c075796.png
+---
+
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { RegionBar } from '@faststore/ui'
+
+import path from 'path'
+import { useSSG } from 'nextra/ssg'
+import { getComponentPropsFrom } from 'site/components/utilities/propsSection'
+
+export const getStaticProps = () => {
+  const regionBarPath = path.resolve(__filename)
+  const components = ['RegionBar.tsx']
+  const [regionBarProps] = getComponentPropsFrom(regionBarPath, components)
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: { regionBarProps },
+    },
+  }
+}
+
+export const RegionBarPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const { regionBarProps } = useSSG()
+  return {
+    RegionBar: <PropsSection propsList={regionBarProps} />,
+  }[component]
+}
+
+<header>
+
+# Region Bar
+
+The Region Bar is a similar proposal for the Region Button, usually used on mobile devices only. It should trigger the RegionModal.
+
+The component is part of [Region](/components/region) feature.
+
+</header>
+
+## Overview
+
+<OverviewSection>
+  <RegionBar onButtonClick={() => {}} />
```

Comment:
Can you add some kind of trigger in this event, like an alert() or something. Just to exemplify the click behavior. When we add the RegionModal should we integrate these components?

Reply:
okay, lemme think of something.

> When we add the RegionModal should we integrate these components?
I don't think we will add here. We will add it on `Region` feature doc page. Sounds good?